typings/objection/index.d.ts

```diff
@@ -352,6 +374,10 @@ declare namespace Objection {
     (column: string, amount?: number): QB;
   }
 
+  interface CountMethod<QB extends AnyQueryBuilder> {
+    (): QB;
```

Comment:
`count` can also take a `ColumnRef` or an array of them. http://knexjs.org/#Builder-count

Reply:
And apparently all kinds of other stuff I wasn't even aware of :smile: